src/Plugin.php

```diff
@@ -245,6 +245,15 @@ public static function loadTextdomain() {
     load_plugin_textdomain(static::L10N, FALSE, static::L10N . '/languages/');
   }
 
+  /**
+   * Returns the plugin meta data.
+   *
+   * @todo Implement caching.
+   */
+  public static function get_plugin_data() {
```

Comment:
```suggestion
  public static function getMetaData() {
```

However, retrieving this at runtime is pretty slow; the function should normally be used in the admin only (or cached appropriately).

I do not actually think that the custom plugin version is necessary, because #29 is actually about to add the current Git commit hash as version number to all assets using the "default version" (the WP Core version) for cache busting.

Therefore, I'd recommend to simply remove this part and just ensure that it is using the default version instead (which I believe it does by default).

Reply:
I was suggesting this to @colourgarden but also mentioned we have the git versioning already but we're not using this in the core standards plugin currently, so we thought of introducing this. I doubt the git versioning will work for plugins published on the official WP plugin repository (haven't checked), so getting the plugin version should be more stable, no?

By default it'll use the current WP version which will not be sufficient to invalidate the assets.